scripts/deploy.sh

```
@@ -0,0 +1,438 @@
+    fi
+    sleep 1
+  done
+  # Hard fallback: if still active after polling, delete and wipe before uploading.
+  _PLUGINS_HTTP=$(curl -s -o /tmp/_plugins.json -w "%{http_code}" \
+    --unix-socket "$_LOCAL_SOCKET" "http://localhost/api/v4/plugins" 2>/dev/null || echo "000")
+  if [[ "$_PLUGINS_HTTP" == "200" ]] && \
+      jq -e ".active[]? | select(.id == \"${PLUGIN_ID}\")" /tmp/_plugins.json > /dev/null 2>&1; then
+    echo "  plugin still active after 10s — forcing removal before upload."
+    curl -s -o /dev/null --unix-socket "$_LOCAL_SOCKET" \
+      -X DELETE "http://localhost/api/v4/plugins/$PLUGIN_ID" || true
+    rm -rf "$_ABS_PLUGIN_DIR/$PLUGIN_ID" 2>/dev/null || true
```

Shellcheck (0.11.0)

[warning] Use "${var:?}" to ensure this never expands to / .

(SC2115)